cmd/main.go

```diff
@@ -81,10 +81,12 @@ func main() {
         "HTTP URL for AWS SQS to connect to; usually best left undefined, "+
             "unless required for local testing purposes (LocalStack uses http://localhost:4566)")
     var eventsTopic = flag.String("events", "", "If defined, it will attempt to connect "+
-        "to the given SQS Queue (ignores any value that is passed via the -kafka flag)")
-    var dlqTopic = flag.String("errors", "",
-        "The name of the Dead-Letter Queue ("+"DLQ) in SQS to post errors to; if not "+
-            "specified, the DLQ will not be used")
+        "to the given SQS Queue to receive events from the Pub/Sub system")
+    // FIXME: this is no longer ONLY for errors, but a more general `notifications` queue,
+    //  so should be renamed accordingly to `-notifications`
+    var notificationsTopic = flag.String("errors", "",
+        "The name of the notification topic in SQS to publish events' outcomes to; if not "+
+            "specified, no outcomes will be published")
```

Comment:
or break it down into two different queues?

Reply:
it would make it unnecessarily complex - in fact, now that I think about this, the change is pretty self-contained to the startup script, so I'll just fix it - no "external" changes will be required.

Reply:
Well, monitoring wise, having the former queue be something we could monitor on errors/bad transitions was useful in that we could alert on _any_ messages in the queue. No other advantages though. Now, the consumer lambda will need to notify on issues it sees in the messages, which is doable.

Reply:
done :+1: